src/components/CheckMarkCard/CheckMarkCard.jsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import getValidProps from '@helpscout/react-utils/dist/getValidProps'
+import { classNames } from '../../utilities/classNames'
+import { createUniqueIDFactory } from '../../utilities/id'
+import { noop } from '../../utilities/other'
+import Icon from '../Icon'
+import Checkbox from '../Checkbox'
+import VisuallyHidden from '../VisuallyHidden'
+import { CheckMarkCardUI, CheckMarkUI } from './CheckMarkCard.css'
+
+const uniqueID = createUniqueIDFactory('CheckMarkCard')
+
+export class CheckMarkCard extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      id: props.id || uniqueID(),
+    }
+
+    this.checkMarkCardRef = React.createRef()
+  }
+
+  componentDidMount() {
+    if (this.props.isFocused && this.inputNode) {
+      this.checkMarkCardRef.current.classList.add('is-focused')
+      this.inputNode.focus()
+    }
+  }
+
+  getClassName() {
+    const { className, checked, disabled } = this.props
+
+    return classNames(
+      'c-CheckMarkCard',
+      className,
+      checked && 'is-checked',
+      disabled && 'is-disabled'
+    )
+  }
+
+  setInputNodeRef = node => {
+    this.inputNode = node
+    this.props.inputRef(node)
```

Comment:
@tinkertrain Just so I can understand this pattern...why would we need both a `inputRef` and an `innerRef` here?

Reply:
No reason! I will remove one, not sure why RadioCard has it this way, after refactoring and removing several thing from there this one escaped me and I brought it over, I'll fix it in both components